homeassistant/helpers/script.py

```diff
@@ -719,6 +720,7 @@ async def async_run_sequence(iteration, extra_msg=""):
         else:
             del self._variables["repeat"]
 
+    @trace_path("choose")
```

Comment:
This means that `default` will also get a `choose` added to the path. That's not correct. https://www.home-assistant.io/docs/scripts/#choose-a-group-of-actions